src/modelgauge/prompt_sets.py

```
@@ -0,0 +1,37 @@
+from modelgauge.secret_values import RequiredSecret, SecretDescription
+
+
+class ModellabFileDownloadToken(RequiredSecret):
+    @classmethod
+    def description(cls) -> SecretDescription:
+        return SecretDescription(
+            scope="modellab_files",
+            key="token",
+            instructions="Please ask MLCommons admin for permission.",
+        )
+
+
+# file name format:
+# {prefix}_{version}_{type}(_{locale})_prompt_set_release
+PROMPT_SETS = {
+    "practice": "airr_official_1.0_practice_prompt_set_release",
+    "official": "airr_official_1.0_heldback_prompt_set_release",
+    "practice_fr_fr": "airr_official_1.0_practice_fr_fr_prompt_set_release",
+}
+TEST_PROMPT_SETS = {
+    "fake-prompts": "fake-prompts",
+}
+
+PROMPT_SET_DOWNLOAD_HOST = "ailuminate.mlcommons.org"
+
+
+def prompt_set_file_base_name(prompt_set: str) -> str:
+    filename = PROMPT_SETS.get(prompt_set, TEST_PROMPT_SETS.get(prompt_set, ""))
```

Hm, maybe this should loudly fail if it gets an unknown prompt set instead of returning an empty string. I think it might be hard to debug otherwise. Or maybe it could return None instead? So that its more obvious that there's not corresponding file name for the prompt set?